### ISSUE TITLE
Checkin confirmation broken for now

### ISSUE THREAD

**Issue by pierrelefevre** [pierrelefevre/ticketmaestro#3]
https://github.com/ethers-io/ethers.js/issues/4513

Seems it's a metamask issue, but should not affect the demo we can just check it on etherscan

**Comment by kule1163** [comment]
Yes i think it is metamask issue because i try with coinbase wallet everything works fine
